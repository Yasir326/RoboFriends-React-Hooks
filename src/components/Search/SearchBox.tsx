import React from 'react';

interface ISearchChange {
  searchChange(event: React.SyntheticEvent<HTMLInputElement>): void
}

const SearchBox = ({ searchChange }: ISearchChange) => {
  return (
    <div className='pa2'>
      <input
        aria-label='Search Robots'
        className='pa3 ba b--green bg-lightest-blue tc'
        type='search'
        placeholder='Search Robots'
        onChange={searchChange}
      />
    </div>
  );
};

export default SearchBox;
