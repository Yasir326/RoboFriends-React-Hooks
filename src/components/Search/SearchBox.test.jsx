import React from 'react';
import { shallow } from 'enzyme';
import SearchBox from './SearchBox';
import '../../setUpTest';

it('expect to render Card component', () => {
  expect(shallow(<SearchBox />)).toMatchSnapshot();
});