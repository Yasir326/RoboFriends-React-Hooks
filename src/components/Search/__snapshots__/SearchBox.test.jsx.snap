// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card component 1`] = `
<div
  className="pa2"
>
  <input
    aria-label="Search Robots"
    className="pa3 ba b--green bg-lightest-blue tc"
    placeholder="Search Robots"
    type="search"
  />
</div>
`;
